diff --git a/.gitignore b/.gitignore
index 30e8234..7b5795c 100644
--- a/.gitignore
+++ b/.gitignore
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+/.yarn
 
 # testing
 /coverage
diff --git a/app/About/page.tsx b/app/About/page.tsx
index d0c649c..3760d42 100644
--- a/app/About/page.tsx
+++ b/app/About/page.tsx
@@ -17,7 +17,7 @@ import {
 } from "@react-spring/web";
 import Hypertext from "../../components/hypertext";
 
-const About: NextPage = () => {
+const About = () => {
   const [mousePos, setMousePos] = useState({ x: 0, y: 0 });
   const leftRef = useSpringRef();
   const upRightRef = useSpringRef();
